src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -70,9 +70,9 @@ public void execute_validCommand_success() throws Exception {
 
     @Test
     public void execute_storageThrowsIoException_throwsCommandException() {
-        // Setup LogicManager with JsonAddressBookIoExceptionThrowingStub
-        JsonAddressBookStorage addressBookStorage =
-                new JsonAddressBookIoExceptionThrowingStub(temporaryFolder.resolve("ioExceptionAddressBook.json"));
+        // Setup LogicManager with JsonExpenseListIoExceptionThrowingStub
+        JsonExpenseListStorage addressBookStorage =
+                new JsonExpenseListIoExceptionThrowingStub(temporaryFolder.resolve("ioExceptionAddressBook.json"));
```

Comment:
75 should this be `new JsonExpenseListIoExceptionThrowingStub(temporaryFolder.resolve("ioExceptionExpenseList.json")); instead?`